src/SockPerf.cpp

```diff
@@ -3346,7 +3343,14 @@ void do_test()
 	info.fd_min = s_fd_min;
 	info.fd_max = s_fd_max;
 	info.fd_num = s_fd_num;
-
+#ifdef USING_VMA_EXTRA_API
+	if (s_user_params.is_vmazcopyread && g_vma_api) {
```

Comment:
check for `s_user_params.is_vmazcopyread` means nothing in case `g_vma_api` is NULL.